src/Features/OnboardingAutomateTaxes.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Onboarding - set up automated tax calculation.
+ *
+ * @package Woocommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Features;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks;
+
+/**
+ * This contains logic for setting up shipping when the profiler completes.
+ */
+class OnboardingAutomateTaxes {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_action(
+			'woocommerce_onboarding_profile_completed',
+			array(
+				__CLASS__,
+				'on_onboarding_profile_completed',
+			)
+		);
+	}
+
+	/**
+	 * Set up automated taxes.
+	 */
+	public static function on_onboarding_profile_completed() {
+		$jetpack_connected = null;
```

Comment:
These checks are based on similar checks we do [here](https://github.com/woocommerce/woocommerce-admin/blob/main/src/Features/ShippingLabelBanner.php#L56). I'd be happy to consider extracting this if I could get some advice on where that logic could belong.

Reply:
It's similar but different. I don't think it's worth extracting just yet. I think if you were to extract it `src/Features/Onboarding.php` would be the place for it.